ironfish-cli/src/commands/wallet/export.ts

```diff
@@ -45,6 +51,12 @@ export class ExportCommand extends IronfishCommand {
     const client = await this.sdk.connectRpc(local)
     const response = await client.exportAccount({ account })
 
```

Comment:
I think instead of modifying the response inplace  it would be better to create a separate RPC (ie `exportAccountBase58`), or just extend the  RPC to take an input of `format` and pass in base58 (or hex as default). 

I know we talked about keeping things in the CLI layer, but it seems like the code will get more cruft if we do that here because of the way the response is used. Also we get the benefit of our RPC being exposed to other wallet developers

Reply:
Got it! 👍